Dockerfile

```diff
@@ -135,14 +133,14 @@ RUN set -x; yum -y update && \
 		yum -y install git systemd \
 		autoconf libtool libcurl-devel libtool-ltdl-devel openssl-devel yajl-devel \
 		gcc gcc-c++ make cmake bison flex file systemd-devel zlib-devel gtest-devel rpm-build systemd-rpm-macros java-11-openjdk-devel \
-		expect rpm-sign zip
+		expect rpm-sign zip wget
```

Comment:
`curl` is usually pre-installed. Do we also need `wget`?

Reply:
There were one or two images that didn't have `curl`. I'm fine just adding `curl` to those if that's preferable over adding `wget` everywhere.

Reply:
I usually prefer `curl` over `wget`…

Reply:
Removed `wget` and added `curl` when needed.